### PR TITLE
Tmo spectrometer

### PR DESCRIPTION
**Pull request by tongju12**
## Description
Change TMO spectrometer new two motors name and position on the GUI
Change name from lens_pitch to lens_pitch_up_down; lens_yaw to lens_yaw_left_right

## Motivation and Context
TMO scientist Mingfu asked for this

## How Has This Been Tested?

```python
In [1]: from pcdsdevices.spectrometer import TMOSpectrometer

In [2]: lx=TMOSpectrometer('TMO:SPEC',name='sp1k4')

In [3]: lx.lens_yaw_left_right.get()
Out[3]: BeckhoffAxisTuple(user_readback=8.400256, user_setpoint=8.400257, user_offset=0.0, user_offset_dir=0, offset_freeze_switch=0, set_use_switch=0, velocity=0.5, acceleration=0.1, motor_egu='mm', motor_is_moving=0, motor_done_move=1, high_limit_switch=0, low_limit_switch=0, high_limit_travel=9.2, low_limit_travel=7.2, direction_of_travel=0, motor_stop=0, disabled=0, description='Main.M42 / Thorlabs2', dial_position=8.400256, plc=BeckhoffAxisPLCTuple(status='', err_code=0, cmd_err_reset=0, cmd_home=0, home_pos=0.0, user_enable=1), motor_spmg=3)

In [4]: lx.lens_pitch_up_down.get()
Out[4]: BeckhoffAxisTuple(user_readback=10.499157, user_setpoint=10.499158, user_offset=0.0, user_offset_dir=0, offset_freeze_switch=0, set_use_switch=0, velocity=0.5, acceleration=0.1, motor_egu='mm', motor_is_moving=0, motor_done_move=1, high_limit_switch=0, low_limit_switch=0, high_limit_travel=11.7, low_limit_travel=7.7, direction_of_travel=0, motor_stop=0, disabled=0, description='Main.M41 / Thorlabs1', dial_position=10.499157, plc=BeckhoffAxisPLCTuple(status='', err_code=0, cmd_err_reset=0, cmd_home=0, home_pos=0.0, user_enable=1), motor_spmg=3)
```

## Where Has This Been Documented?
This PR

## Pre-merge checklist
- [x] Code works interactively
- [ ] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on travis
- [ ] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

File: docs/source/upcoming_release_notes/1068-changing_sp1k4_two_new_motors_name.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+1068 changing sp1k4 two new motors name
+#################
```
Comment (tangkong) [inline docs/source/upcoming_release_notes/1068-changing_sp1k4_two_new_motors_name.rst:2]:
```suggestion
1068 changing sp1k4 two new motors name
#######################################
```

File: docs/source/upcoming_release_notes/1068-changing_sp1k4_two_new_motors_name.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+lens_pitch to lens_pitch_up_down
+lens_yaw to lens_yaw_left_right
```
Comment (tangkong) [inline docs/source/upcoming_release_notes/1068-changing_sp1k4_two_new_motors_name.rst:15]:
```suggestion
- Renames ``lens_pitch`` to ``lens_pitch_up_down`` and ``lens_yaw`` to ``lens_yaq_left_right``
```

File: docs/source/upcoming_release_notes/1068-changing_sp1k4_two_new_motors_name.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+--------
+- N/A
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+tongju@slac.stanford.edu
```
Comment (tangkong) [inline docs/source/upcoming_release_notes/1068-changing_sp1k4_two_new_motors_name.rst:31]:
```suggestion
- tongju12
```

**Review by tangkong — COMMENTED** [review]
This looks fine to me.  I added a few formatting suggestions on the pre-releaase notes, but aside from that I think this should be ok to merge.

File: docs/source/upcoming_release_notes/1068-changing_sp1k4_two_new_motors_name.rst (hunk 3 above)
Comment (tongju12) [reply docs/source/upcoming_release_notes/1068-changing_sp1k4_two_new_motors_name.rst:31]: I accidentally close this pull request and reopen again. Sorry about it. Can you approve again? @tangkong

**Review by ZLLentz — APPROVED** [review]

**Comment by ZLLentz** [comment]
I'll bring this into the ui dev

**Pull request merged**